pandas/core/frame.py

```diff
@@ -1018,8 +1018,8 @@ def itertuples(self, index=True, name="Pandas"):
         # use integer indexing because of possible duplicate column names
         arrays.extend(self.iloc[:, k] for k in range(len(self.columns)))
 
-        # Python 3 supports at most 255 arguments to constructor
-        if name is not None and len(self.columns) + index < 256:
+        # Python versions before 3.7 support at most 255 arguments to constructor
```

Comment:
is it possible to skip the arg len check for >= 3.7? does it work?

Reply:
yes, this was simple to do.